wger/core/api/serializers.py

```diff
@@ -16,14 +16,36 @@
 # along with Workout Manager.  If not, see <http://www.gnu.org/licenses/>.
 
 from rest_framework import serializers
-
+from django.contrib.auth.models import User
+from rest_framework.validators import UniqueValidator
 from wger.core.models import (
     UserProfile,
     Language,
     DaysOfWeek,
     License,
     RepetitionUnit,
-    WeightUnit)
+    WeightUnit,
+    ApiUser)
+
+
+class UserSerializer(serializers.ModelSerializer):
+    """ Serializer to map to User model """
+    email = serializers.EmailField(required=True,
+                                   validators=[UniqueValidator(queryset=User.objects.all())])
+
+    username = serializers.CharField(validators=[UniqueValidator(queryset=User.objects.all())])
+
+    password = serializers.CharField(min_length=8, write_only=True)
+
+    def create(self, validated_data):
+        user = User.objects.create_user(validated_data['username'], validated_data['email'],
+                                        validated_data['password'])
```

Comment:
from from the documentation the user model hashes the password.
when i check the database , the passwords are hashed